### PR TITLE
Add case for validation when country has no provinces

### PR DESCRIPTION
**Pull request by serene**

### CODE REVIEW

File: lib/ama_validators/province_validator.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 class ProvinceValidator < ActiveModel::EachValidator
   def validate_each(object, attribute, value)
     provinces = PROV_STATE[object.send(options[:country]).to_sym]
-    if !provinces.try(:include?, value)
+    if provinces.present? && !provinces.try(:include?, value)
```
Comment (MathieuGilbert) [inline lib/ama_validators/province_validator.rb]: If you're checking for presence first, the `.try` isn't needed.

**Comment by MathieuGilbert** [comment]
:hankey:

**Comment by serene** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**